frontend/packages/console-shared/src/components/synced-editor/index.tsx

```diff
@@ -42,7 +42,11 @@ export const SyncedEditor: React.FC<SyncedEditorProps> = ({
   const { formContext, yamlContext } = context;
   const { t } = useTranslation();
   const [formData, setFormData] = React.useState<K8sResourceKind>(initialData);
-  const [yaml, setYAML] = React.useState(safeJSToYAML(initialData));
+  const [yaml, setYAML] = React.useState(
+    safeJSToYAML(initialData, 'yamlData', {
+      skipInvalid: true,
```

Comment:
Added the `skipInvalid` property to parse object with undefined value similar to how it is used in line 80
https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/components/synced-editor/index.tsx#L80